RestApi/Sepes.RestApi/Model/PodInput.cs

```diff
@@ -3,10 +3,17 @@ namespace Sepes.RestApi.Model
 
     public class PodInput
     {
-        public string podName { get; set; }
-        public int studyID { get; set; }
-        public string podTag { get; set; }
-        public int podID { get; set; }
+        public int id { get; set; }
```

Comment:
These changes are out of scope.
It's likely that frontend also must be changed based on this because it must use the same names.

Reply:
I will revert these changes. Its true they where out of scope